client/js/components/faq/DpFaqSupport.vue

```diff
@@ -0,0 +1,121 @@
+<license>
+(c) 2010-present DEMOS plan GmbH.
+
+This file is part of the package demosplan,
+for more information see the license file.
+
+All rights reserved
+</license>
+
+<template>
+  <div
+    id="scFaq"
+    class="c-support__wrapper space-inset-m">
+    <h2 class="font-normal">
+      {{ Translator.trans('support.heading') }}
+    </h2>
+    <section class="mb-5">
+      <p class="mb-5">
+        {{ Translator.trans('support.introduction') }}
+      </p>
+      <p>
+        Sie haben eine Frage zur Technik? Dann hilft Ihnen unser technischer Support gerne weiter.
+      </p>
+        <h3 class="font-size-h4 mt-5">
```

Comment:
why do you need `font-size-h4` class here? The `h3` heading already has a font size of 24px, which is almost the same as in the design template